vitest.config.ts

```diff
@@ -1,6 +1,12 @@
+import react from '@vitejs/plugin-react'
 import {defineConfig} from 'vitest/config'
 
 export default defineConfig({
+  plugins: [
+    react({
+      babel: {plugins: [['babel-plugin-react-compiler', {target: '18'}]]},
+    }),
+  ],
```

Comment:
It's vital that the test suite is also using the react compiler, to ensure we're testing the behaviour of the code we're shipping, not the unoptimised code.
Especially since our testing suite is testing implementation details around memoization.

Reply:
💯